ttg/ttg/terminal.h

```diff
@@ -230,6 +292,19 @@ namespace ttg {
       return get_connections();
     }
 
+    template <typename Key = keyT, typename Value = valueT>
+    std::enable_if_t<meta::is_none_void_v<Key,Value> && std::is_same_v<Value,std::remove_reference_t<Value>>,void>
+    send_to(const Key &key, Value &&value, std::size_t i)
```

Comment:
Is this used anywhere?

Reply:
All the unused code is leftover from pure tasks implementation for pull terminals. I will remove it.